omnibus/config/projects/chef.rb

```diff
@@ -45,27 +45,27 @@
 overrides_path = File.expand_path("../../../../omnibus_overrides.rb", current_file)
 instance_eval(IO.read(overrides_path), overrides_path)
```

Comment:
Let's remove this line and remove `the `omnibus_overrides.rb` file. The override files have been moved to https://github.com/chef/chef-foundation for chef-16 and chef-18